data/com.github.Alcaro.Flips.Source.svg

```diff
@@ -0,0 +1,3040 @@
+           id="rect978"
+           width="112"
+           height="114"
+           x="-674.01123"
+           y="357.5513"
+           rx="56"
+           ry="56" />
+        <path
+           style="display:inline;opacity:1;fill:#f6f5f4;fill-opacity:1;stroke:none;stroke-width:0.01129822px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;marker:none;marker-start:none;marker-mid:none;marker-end:none;paint-order:normal;enable-background:new"
+           d="m -558.01133,413.5513 a 59.999995,59.999995 0 0 1 -59.99999,59.99999 59.999995,59.999995 0 0 1 -60,-59.99999 59.999995,59.999995 0 0 1 60,-60 59.999995,59.999995 0 0 1 59.99999,60 z"
+           id="path981"
+           inkscape:connector-curvature="0" />
+        <circle
+           r="54"
+           style="display:inline;opacity:1;fill:#424048;fill-opacity:1;stroke:none;stroke-width:0.01129822px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;marker:none;marker-start:none;marker-mid:none;marker-end:none;paint-order:normal;enable-background:new"
+           id="circle983"
+           cx="-618.01129"
+           cy="413.5513" />
+        <path
+           inkscape:connector-curvature="0"
+           id="path928"
+           d="m -591.9764,386.54325 a 1.0016649,1.0016649 0 0 0 -0.77686,0.33008 l -21.97461,23.66504 a 5.9999995,5.9999995 0 0 0 -5.58203,-0.52246 l -14.2207,-15.80079 a 2.0033298,2.0033298 0 0 0 -1.60352,-0.68212 2.0033298,2.0033298 0 0 0 -1.37402,3.36181 l 14.21924,15.79932 a 5.9999995,5.9999995 0 0 0 -0.20997,5.30517 5.9999995,5.9999995 0 0 0 2.4043,2.69922 l -11.8584,30.49366 a 1.0016649,1.0016649 0 1 0 1.86719,0.72558 l 11.85742,-30.49121 a 5.9999995,5.9999995 0 0 0 3.65332,-0.39209 5.9999995,5.9999995 0 0 0 3.03516,-7.92383 5.9999995,5.9999995 0 0 0 -0.7251,-1.2041 l 21.9795,-23.66992 a 1.0016649,1.0016649 0 0 0 -0.69092,-1.69336 z"
+           style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:0.2;isolation:au
```

Comment:
I think it contains lots of useless example icons, I'll drop them.

BTW, do you want to keep the pixelated look of the icon or would you prefer a smooth one with the same design? E.g. I did this for the symbolic initially: 

![com github Alcaro Flips-symbolic](https://user-images.githubusercontent.com/2536339/73135913-3a68ef80-4048-11ea-910d-cbeef468a81d.png)

Of course I have it as a tiny 16x16 SVG, but GitHub didn't let me upload it in a comment. If you like it, I could update the main icon in the same way. I do understand the point of the pixelated style, I'm simplify offering. :p

Reply:
Actually, we don't even *have* to keep the source file. It's easy to recreate it if ever needed from the exported SVGs.

Reply:
I'd rather go with the smoother look. I never saw the icons as pixeled, it's just that anything too detailed looks bad at such a small size. This is also why the smile is so crooked - it's hard to do any better at such a limited pixel budget, but it looks silly at that size. Does Flathub support using different files for different sizes?

I must also insist that the leaves remain. The moon alone is Lunar IPS, let's not steal its identity.

Reply:
What about this? I went with a more sonic-like style for the leaves (mostly because it was easier, but also I think it looks cool).

![icons](https://user-images.githubusercontent.com/2536339/73141110-8c2d6c00-4080-11ea-90ac-876b4389908d.png)

Reply:
That's ... interesting. Not what I'm thinking of, but not bad. I'll probably tweak it a bit at some point, but it's good enough for this PR.

Or maybe I'll make that the official logo and change it on Windows too, haven't decided yet. Just pasting two SMW objects together is fairly unimaginative.

(Remember to add the leaves to the black one.)

Reply:
![flips-symbolic](https://user-images.githubusercontent.com/2536339/73141900-41fcb880-4089-11ea-86e2-5b5dc1467fac.png)

A preference? Keep in mind it's designed for 16x16 mainly.

Reply:
Of the big ones, middle. Of the smaller ones, all three look as good as SVGs usually do at that size - not very.

I'm pretty sure GNOME supports using different files for different sizes; if Flatpak does too, let's do that and add some hand-edited PNGs for the smaller sizes.

This would also allow us to replace and simplify the icon depending on its size, like on Windows.
![flips-0](https://user-images.githubusercontent.com/5085186/73142159-e253dc80-408b-11ea-88a8-25438c3b7ad0.png) ![flips-1](https://user-images.githubusercontent.com/5085186/73142160-e2ec7300-408b-11ea-86e3-66849e545f2e.png) ![flips-2](https://user-images.githubusercontent.com/5085186/73142161-e2ec7300-408b-11ea-8b0f-15e3122f445a.png)
(You can extract them with, for example, ImageMagick `convert flips.ico flips.png`.)

Reply:
I did something slightly different in the end, which I think will work better: ![com github Alcaro Flips-symbolic svg](https://user-images.githubusercontent.com/2536339/74083041-e99dc180-4a5f-11ea-9d95-0d356af901e2.png)

Indeed Linux desktops support multiple sizes, but it's strongly recommended nowadays to come up with only two icons:
- a 128x128 full color SVG
- a 16x16 symbolic SVG

Any other sizes and matricial formats are now strongly discouraged.